src/xFactor/Judge.java

```diff
@@ -0,0 +1,104 @@
+package xFactor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Judge extends Person {
+	private static int maxFavourites;
+	private static int numberOfJudges;
+	private static int countInstances;
+	private static ArrayList<Judge> judges = new ArrayList<Judge>(numberOfJudges);
+	private int vote;
+	private boolean isSpecial;
+	private ArrayList<Participant> favourites = new ArrayList<Participant>(maxFavourites);
+	private HashMap<Participant, Integer> votes = new HashMap<Participant, Integer>();

```

Comment:
What will happen when you go the the next stage of the competition? You will already have the participants and votes for them.
Why you should have the same information among all judges?